src/ontology/mondo-edit.obo

```diff
@@ -179367,19 +179367,23 @@ synonym: "Johanson-Blizzard syndrome" EXACT [MONDO:Lexical, OMIM:243800]
 synonym: "Johanson-BLIZZARD syndrome; JBS" RELATED [OMIM:243800]
 synonym: "nasal alar hypoplasia, hypothyroidism, pancreatic achylia and congenital deafness" RELATED [GARD:0000080]
 synonym: "nasal alar hypoplasia, hypothyroidism, pancreatic achylia, and congenital deafness" RELATED [OMIM:243800]
+synonym: "pancreatic insufficiency, combined exocrine" EXACT [OMIM:260450]
 xref: DOID:14694 {source="MONDO:equivalentTo"}
 xref: EFO:0001063 {source="MONDO:equivalentTo"}
 xref: GARD:0000080 {source="MONDO:equivalentTo"}
 xref: ICD10:Q87.8 {source="ORDO:2315/attributed", source="ORDO:2315/ntbt", source="Orphanet:2315"}
 xref: ICD9:759.89 {source="MONDO:relatedTo", source="i2s"}
 xref: MESH:C535880 {source="MONDO:equivalentTo", source="Orphanet:2315", source="ORDO:2315/e", source="DOID:14694", source="MONDO:ontobio"}
+xref: MESH:C564907 {source="MONDO:equivalentTo", source="MONDO:ontobio"}
 xref: MESH:D003638 {source="EFO:0001063"}
 xref: NCIT:C27644 {source="EFO:0001063"}
 xref: OMIM:243800 {source="MONDO:equivalentTo", source="Orphanet:2315", source="ORDO:2315/e", source="DOID:14694", source="GARD:0000080", source="EFO:0001063"}
+xref: OMIM:260450 {source="MONDO:equivalentTo"}
```

Comment:
you are proxy merging two OMIMs. Is this deliberate?

Reply:
yes, OMIM deprecated OMIM:260450 and merged it with OMIM:243800

see: https://www.omim.org/entry/260450